src/convertToObject.js

```diff
@@ -13,6 +13,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringToArr = sourceString.split(':').join(';').split(';')
+    .map(item => item.trim());
+  const cssRules = {};
+
+  for (let i = 0; i < stringToArr.length; i++) {
+    if (stringToArr[i] !== '') {
+      cssRules[stringToArr[i]] = stringToArr[i + 1];
+      i++;
```

Comment:
if you need to increase `i` by 2, you can do it inside `for` parameters
![image](https://user-images.githubusercontent.com/35385361/116283194-46d08080-a794-11eb-8bd8-5095c4083737.png)